src/lib/flow/Kconfig

```diff
@@ -13,7 +13,7 @@ config NODE_DESCRIPTION
 
 config RESOLVER_CONFFILE
 	bool "Resolver conffile"
-	depends on NODE_DESCRIPTION
+	depends on NODE_DESCRIPTION && HAVE_GLIB
```

Comment:
Ok, but can't we then submit a patch to remove the dependency instead of adding it back again?